onnxruntime/core/dlpack/dlpack_converter.cc

```diff
@@ -197,10 +197,23 @@ bool IsContiguousTensor(const DLTensor& tensor) {
 
 }  // namespace
 
+static void DlpackCapsuleDestructor(PyObject* data) {
```

Comment:
Yes we need, looks they move it from onnxruntime/python/onnxruntime_pybind_state.cc  to onnxruntime/python/onnxruntime_pybind_state_common.cc . 

We put those dlpack related things in core, because our code written in core onnxruntime lib need reference them, not only the pybind parts.

Reply:
So what we do here is, move the dlacks from onnxruntime/python/onnxruntime_pybind_state_common.cc  to onnxruntime/core/dlpack/dlpack_converter.cc

Reply:
@baijumeswani does this impact the recent changes to ORTValueVector pybinds? I see build failure related to undefined reference related to DLPack

Reply:
I don't think this impacts the changes related to ```ORTValueVector``` as long as ```ORTValueVector``` is able to reference this destructor function. My only concern would be to this relies on the python/c api which in my opinion would be better off in the pybind source files.
But if this move is necessary, I think we should be ok.